README.md

```diff
@@ -65,6 +65,7 @@ potentially very difficult to debug due to dissimilar or unsupported package ver
     - [Composer](#composer-autoloader)
     - [Composer Plugins](#composer-plugins)
     - [PSR-0 Partial support](#psr-0-partial-support)
+    - [WordPress](#wordpress)
```

Comment:
Is it a limitation though?

I realise there is a mistake above in the ToC: Phive, PHAR and Composer should be under "installation" not "usage".  And maybe you can create a new ToC entry under "usage", e.g. "Wordpress integration"